src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,29 @@
+.catalog {
+  display: grid;
+  grid-template-columns: $card-width;
+  justify-content: center;
+  gap: $card-gap;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
+
+  &__card {
+    display: flex;
```

Comment:
![image](https://user-images.githubusercontent.com/70009205/207388118-fd571850-8745-4dc4-903f-f3cce631f4ce.png)
replace to grid, please